feature_engine/variable_manipulation.py

```diff
@@ -93,6 +93,20 @@ def _find_or_check_numerical_variables(
     return variables
 
 
+def contains_categorical(column: pd.Series) -> bool:
```

Comment:
maybe `is_categorical_variable()`, represents more what the function is doing. What do you think?